cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -229,7 +225,14 @@ private void finishInitialization(Bundle savedInstanceState) {
     }
     // Set to overlay video mode.
     mShellManager.getContentViewRenderView().setOverlayVideoMode(true);
-    mShellManager.launchShell(shellUrl);
+
+    // Load an empty page to let shell create WebContents.
+    mShellManager.launchShell("");
+    // Inject JavaBridge objects to the WebContents.
+    initializeJavaBridge();
+    // Load the url with the same shell we created above.
```

Comment:
nit: ` // Load the |url|` or `// Load the `` url `` ` (this latter just with one backtick, markdown is doing things to this text).